orgs/beta_prerelease.json

```
@@ -0,0 +1,14 @@
+{
+  "orgName": "NPSP - Beta Test Org",
```

```suggestion
  "orgName": "NPSP - Prerelease Beta Test Org",
```

(so that it can be distinguished when looking at ScratchOrgInfo objects in the dev hub)